TEST_GUIDE.md

```diff
@@ -30,7 +33,7 @@ Click **Ok** to dismiss the dialog.
 
 ## Install the firmware
 
-These releases have the corresponding firmware bundled in. You no longer need to specify which machine  you have for the firmware.
+These releases have the corresponding firmware bundled in. You no longer need to specify which machine  you have for the firmware. This is with the exception of Linux, where you need to compile and drop in the firmware to upload it through AYAB Desktop (or upload it through other means - described above in the Extra Linux setup section).
```

Comment:
All releases (also the AppImage for Linux) should contain a firmware file (if this is currently not the case, then it's a bug :( )
The need to put a firmware file there manually is only when building from source, no matter which OS (see comment above)

Reply:
Yeah the tar.gz in the release does not contain the firmware hex file - was asked on Discord to just add this to the instructions to make testing a bit easier for whoever works on this next.

Reply:
Or to be honest, running the AppImage failed for me with `./linux-build/appimage/AppRun: line 14: opt/python3.6/bin/fbs: No such file or directory` so I opted to run it with `fbs run` with the hex file dropped into the right location.